whirlpool/appliance.py

```diff
@@ -25,16 +26,20 @@ def __init__(
         backend_selector: BackendSelector,
         auth: Auth,
         said: str,
-        attr_changed: Callable,
     ):
         self._backend_selector = backend_selector
         self._auth = auth
         self._said = said
-        self._attr_changed = attr_changed
+        self._attr_changed: list(Callable) = []
+
         self._data_dict = None
 
         self._session: aiohttp.ClientSession = None
 
+    def register_callback(self, update_callback: Callable):
```

Comment:
```suggestion
    def register_attr_callback(self, update_callback: Callable):
```
Just to make it more clear in case we add other callbacks in the future for other stuff